src/main/java/org/proshin/finapi/user/FpUsers.java

```diff
@@ -48,7 +48,13 @@ public User authorized() {
     @Override
     public boolean verified(final String userId) {
         return new JSONObject(
-            this.endpoint.get(this.url + "verificationStatus", this.token)
+            this.endpoint.get(
+                this.url + "verificationStatus",
+                this.token,
+                new IterableOf<>(
```

Comment:
Currently the project fails to build due to an issue with imports. Please, add the following imports:
```java
import org.cactoos.iterable.IterableOf;
import org.proshin.finapi.primitives.pair.UrlEncodedPair;
```